$OpenBSD: patch-src_thread_c,v 1.3 2010/03/29 12:16:08 dcoppa Exp $

fix from upstream svn:
when dontplaynext is set, request next item on 
playlist from plugin

--- src/thread.c.orig	Thu Feb 25 17:01:41 2010
+++ src/thread.c	Wed Mar 24 12:07:32 2010
@@ -1496,6 +1496,13 @@ gpointer launch_player(gpointer data)
                 g_idle_add(play, p);
             }
 
+			if (playback_error == NO_ERROR) {
+		if (embed_window != 0 || control_id != 0) {
+		    dbus_send_event("MediaComplete", 0);
+		    dbus_open_next();
+		}
+			}
+
             dontplaynext = FALSE;
         }
 
